grunttasks/build.js

```diff
@@ -64,6 +64,8 @@ module.exports = function (grunt) {
     // update the sourcemap path to match the hosted files
     'sourcemap-source',
 
-    'htmlmin',
+    'cdnify',
```

Comment:
could you please add a quick note about what this does as part of this task, just like the ones above:

![](http://v14d.com/i/56b525925db36.jpg)